src/pages/cli/graphql/examples-and-solutions.mdx

```diff
@@ -564,4 +564,75 @@ query repsByPeriodAndTotal {
     }
   }
 }
-```
\ No newline at end of file
+```
+
+## Advanced
+
+### Batch Put Custom Resolver
+
+Sometimes you need to create objects in bulk, rather than creating individual objects sequentially and waiting for all the requests to complete.
+
+1. Define your schema with a custom mutation (the custom mutation should not be deployed to AppSync beforehand if following these steps, the CLI will attach its own resolver preventing you from attaching a custom resource this way)
+```graphql
+type Todo @model @auth(rules: [{allow: private, provider: iam}]) {
```

Comment:
Is the `@auth` part necessary for the example? If not, I'd drop it to keep things focused.

Reply:
Just wanted something that would cause the migration to fail, I can pick a simpler case

Reply:
I take that back, I was thinking of a different PR and forgot where I was